utils/mp-server/mp-query-scripts/top-10-maps-by-game-count.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+OUTFILE=top-10-maps-by-game-count.csv
+echo "MAP_NAME,MAP_COUNT" > $OUTFILE
+mysql -s -e "select MAP_NAME, count(*) as MAP_COUNT \
+from wesnothd_game_info \
+where YEAR(START_TIME) = YEAR(CURRENT_DATE - INTERVAL 1 MONTH) \
+  and MONTH(START_TIME) = MONTH(CURRENT_DATE - INTERVAL 1 MONTH) \
+  and END_TIME is not NULL \
+  and TIMESTAMPDIFF(MINUTE, START_TIME, END_TIME) > 5 \
+group by MAP_NAME \
+order by count(*) desc \
+limit 10" | sed 's/\t/,/g' >> $OUTFILE
```

Comment:
Could be higher than 10

Reply:
10 was pretty much an arbitrary choice, other than "top 10" lists being commonly done, as well as the full list ends up with a long tail of rows with a count of a few games or less which didn't seem useful at that point.

Reply:
That could be solved by "having count(*) > x".

Reply:
Still would need to decide on an "x" value.  So maybe it would be better to not limit it and let anyone looking filter however they want in Excel or Calc or whichever software they're using.